source/install/requirements.rst

```diff
@@ -116,6 +116,8 @@ In MySQL 8.0.4, the default authentication plugin was changed from ``mysql_nativ
    
    [mysqld]
    default-authentication-plugin=mysql_native_password
+   
+In MySQL versions 8.0.0-8.0.11 ```ADMIN``` is a `reserved keyword <https://dev.mysql.com/doc/refman/8.0/en/keywords.html>`_. We recommend installing 8.0.12 or higher since it is no longer reserved in these versions.
```

Comment:
I think we should not "recommend", but require it. If someone's using v8, the minimum supported version is 8.0.12. Otherwise, they will run into problems. Also, maybe people don't really care about the details of the admin word being a reserved keyword (it's just the implementation detail that makes Mattermost incompatible with 8.0.0-8.0.11)

Reply:
@agarciamontoro I've removed the language around "recommendations" and clarified the requirement around 8.0.4 on updating the plugin portion.  I have chosen to keep the information that explains why we don't support v 8.0.0-8.0.11 so customers have more context around it.